.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module",
+      "ecmaFeatures": {
+          "jsx": true
+      }
+  },
+  "rules": {
+      "semi": "error"
```

Comment:
`jsx` isn't used anywhere in the project, we don't need `2018` edition, and `semi` is already included in the [config](https://github.com/mourner/eslint-config-mourner/blob/master/index.js) used in the project. If you need any overrides, let's keep them in `package.json`: https://github.com/mapbox/geojson-vt/blob/master/package.json#L32